Automattic/wp-calypso
# SSR: Move anything not needed by the masterbar back to `client`

This moves unused dependencies (on the server), back to `client`. It's the first part of #1551, in a later PR, we'll remove `shared` altogether. 

Note that i18n-utils and its dependencies remain in `shared`, as it's used by both the client and server for user bootstrapping.

To test:
- `make run`
- Check there are no build errors
- Check Calypso generally works (the e2e tests are nice for this)
- Check the logged-out masterbar on `/design` works, and is SSRed
